CHANGELOG-3.0.md

```diff
@@ -88,10 +88,22 @@
     - `Bundle\CoreBundle\Helper\BundlesSchemaHelper::createSchema` visibility set to private
     - There is no `web/bootstrap-font-awesome.css` file generated anymore. Instead, Bootstrap and Font Awesome are always included independently.
     - Removed the `%temp_dir%` parameter. If you need a temporary folder use `sys_get_temp_dir()`.
-    - Removed custom translation system (#4042).
+    - Removed custom translation system (#4042). Use Symfony's translation system directly.
     - Removed use of `admin.png` and replaced by adding icon class to `composer.json` >> `extra/zikula/capabilities/admin/icon: "fas fa-user"`
     - Replaced `LinkContainer` with `ExtensionMenu` for collecting module menus (admin, user, account). See companion docs.
     - Setting `composer.json` >> `extra/zikula/capabilities/admin/url` is no longer supported. Use `extra/zikula/capabilities/admin/route`.
+    - Changes regarding directory layout
+        - The `app/config/` directory has been moved to `config/`.
+        - The `app/Resources/<BundleName>/views/` directory is now located at `templates/bundles/<BundleName>/`.
+        - The `app/Resources/` directory became `Resources/`.
```

Comment:
this part seems odd to me (and `Resources/` is not in the best-practices document). With everything else removed from this, I suspect the last remaining directory is `workflows/`? I recommend we eliminate `Resources/` entirely and only have `worklows/` (like `templates`, `translations` etc)

Reply:
It is from https://github.com/symfony/symfony/blob/master/UPGRADE-4.0.md

Reply:
ok - but is workflows better in `src/worflows/` ? and if so, what is left to put in `Resources/`?

Reply:
Removed `Resources` entirely in https://github.com/zikula/core/commit/9732263c64509b471248b44a15b49fecc60b8a3f

custom workflows can also be added by amending `config/packages/workflows.yaml` or possibly even `config/services_custom.yaml`

Reply:
added `src/workflows/`